modules/rosa-upgrading-automatic-ocm.adoc

```diff
@@ -5,16 +5,18 @@
 
 :_content-type: PROCEDURE
 [id="rosa-scheduling-upgrade_{context}"]
-= Scheduling automatic upgrades
+= Scheduling recurring upgrades for your cluster
 
-You can schedule automatic upgrades for a {product-title} cluster through {cluster-manager-first} console.
+You can schedule recurring, automatic upgrades for your {product-title} cluster on a weekly basis through the {cluster-manager} console.
 
 .Procedure
 
 . Log in to {cluster-manager-url}.
 . Select a cluster to upgrade.
 . Click the *Settings* tab.
-. In the *Update strategy* pane, click *Automatic* and select a preferred day of the week and start time for the automatic upgrades.
+. In the *Update strategy* pane, select *Recurring updates*.
```

Comment:
```suggestion
. In the *Update strategy* panel, select *Recurring updates*.
```

Reply:
Throughout the docs and looking at the UI, the UI reference is 'pane'. Here's the guideline in the IBM style guide:
- A pane is a framed section of a window that provides different views of the same document or independent but related information.
- A panel sits flush to either the left or right side. Left panels are used for a product's navigation. Right panels are used for additional system-level actions and are sometimes used for general tasks. 
Not making any change here, unless you think otherwise.

Reply:
TIL, thanks for explanation.